public/pages/CreateTransform/components/PreviewOptions/Panels/EditTransformPanel/EditTransformPanel.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React, { useState } from "react";
+import { EuiButton, EuiFieldText, EuiFlexGroup, EuiFlexItem, EuiFormRow, EuiPanel, EuiSpacer, EuiText } from "@elastic/eui";
+import { TransformAggItem } from "../../../../../../../models/interfaces";
+
+interface EditTransformPanelProps {
+  name: string;
+  aggList: TransformAggItem[];
+  onEditTransformation: (oldName: string, newName: string) => void;
+  closePopover: () => void;
+}
+
+export default function EditTransformPanel({ name, aggList, onEditTransformation, closePopover }: EditTransformPanelProps) {
```

Comment:
Should we add jest tests for these?

Reply:
Yes, I believe the unit tests are currently being written.